### ISSUE TITLE
API endpoint to check user ability to afford products for burger-list

### ISSUE THREAD

**Issue by alexaor** [KSG-IT/ksg-nett#89]
Suggestion to have API endpoints such that whenever we get a tablet-application rolling that one can generate a list of whoever is working, with the ability to modify said list and then do a "check" for that list returning whether or not anyone on the list cannot afford what they are about to purchase for stilletime. This is not to be confused with a purchase-order, the idea is that people can make orders, which are then added to the list, if anyone does not have sufficient funds the tablet-user would get feedback regarding who does not have sufficient funds. The list could then later be "locked" when people have had the opportunity to transfer funds.

TL;DR 
API endpoint that accepts a complete list of people and their respective orders, returns list of boolean values whether or not people have sufficient funds. Then maybe another endpoint for the actual locking and withdrawal of funds if it is not implemented already.
